functions/Test-DbaLinkedServerConnection.ps1

```diff
@@ -0,0 +1,76 @@
+function Test-DbaLinkedServerConnection
+{
+<#
+.SYNOPSIS
+Test all linked servers from the sql servers passed
+
+.DESCRIPTION
+Test each linked server on the instance
+
+.PARAMETER SqlServer
```

Comment:
Super useful command!  I'll change this to SqlInstance for you -- it's really the only thing failing the Pester tests that ensure Get-Help describes all params.